Feature/add game controller

Diegou2304 opened the pull request:
Addition of basic controller for games entity with its basic methods

Diegou2304 committed:
Addition of GameController with its basic methods

Dylan-Uribe commented on Game-Inventory.API/Controllers/Games/GameController.cs:
```diff
@@ -0,0 +1,59 @@
+
+
+[ApiController]
+public class GameController : ControllerBase
```
El nombre del controlador debe estar en plural "GamesController"

Dylan-Uribe commented on Game-Inventory.API/Controllers/Games/GameController.cs:
```diff
@@ -0,0 +1,59 @@
+    [HttpDelete]
+    [Route("/games")]
+    [SwaggerOperation(Tags = ["Games"])]
+    public IActionResult GetGames([FromBody] int id)
```
El nombre de la función GetGames no representa la solicitud que se está haciendo [HttpDelete]

Diegou2304 committed:
Fix comments

Diegou2304 committed:
Fix Comments
Fix Merge conflict
Update branch from latest dev changes
Delete default weatherforecast controller
